### PR TITLE
Bug 1381408, Fix sizing guidelines

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1381408

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@jeremyeder can you please take a look? Thanks!

**Comment by ahardin-rh** [comment]
@jeremyeder says that maximum pods per nodes should still be 250. Not sure how this needs to be updated, but GSS is saying that this section of the docs is incorrect. @timothysc are you able to help? Thanks!

**Comment by timothysc** [comment]
The table with the maximum pods/node is correct.  Perhaps we should add the hint that you need to do the math then.  e.g. @ 250 pods/node you'll top out at 480 nodes.  

Those guidelines are upper bounds, everything is correct.

**Comment by ahardin-rh** [comment]
@adellape Made some updates. Can you PTAL? 

Preview build: http://file.rdu.redhat.com/~ahardin/10262016/sizing-fixes/install_config/install/planning.html#sizing

File: install_config/install/planning.adoc [hunk 1]
```diff
@@ -52,27 +52,50 @@ If you install initially using the quick installer, you can always further tweak
...
+The following table provides the maximum sizing limits for nodes and pods:
 
 [cols="8,2",options="header"]
 |===
 |Type |Maximum
 
 |Maximum nodes per cluster |1000
 
-|Maximum pods per cluster |120,000
+|Maximum pods per cluster |120000
```
Comment (adellape) [inline install_config/install/planning.adoc]: http://stylepedia.net/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-N

See "numbers" for comma usage.

File: install_config/install/planning.adoc [hunk 2]
```diff
@@ -52,27 +52,50 @@ If you install initially using the quick installer, you can always further tweak
...
+----
+
+If you increase the number of nodes to 20, then the pod distribution changes to
+5000 pods per cluster:
+
+----
+20 * 250 = 5000
+----
 
 [IMPORTANT]
```
Comment (adellape) [inline install_config/install/planning.adoc]: Maybe this IMPORTANT note can go up after the table now, but before Scenario A?

**Comment by timothysc** [comment]
Better, but focuses too much on the high end number which is more an edge case. 

Maximum Pods / (Expected Pods/Node <= Max Pods/Node) = Total Number of nodes. 

Admins should profile their pods and nodes to determine exactly how many pods they expect to fit per node.

**Comment by ahardin-rh** [comment]
@timothysc Okay, thanks!

Do you think the scenario approach is okay? If so, do you have any suggested (more realistic/less edge case) scenarios we can use?

Also, is it Scenario A that focuses too much on the high end numbers?

**Comment by ahardin-rh** [comment]
@adellape I made some updates. @timothysc Any suggested scenarios I should use to make the numbers more realistic? Should I just nix the first scenario altogether? Thanks.

**Comment by ahardin-rh** [comment]
Made further updates. Sending to QE for review.

**Comment by ahardin-rh** [comment]
QE verified

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/install/planning.adoc#install-config-install-planning[Installing -> Planning]
|Updated the xref:../install_config/install/planning.adoc#sizing[Sizing Considerations] section for clarity.
%

**Pull request merged**